src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -34,6 +66,14 @@ public IntakeSubsystem() {
     motor = MotorController.constructMotor(MotorConfig.IntakeMotor1);
     motor2 = MotorController.constructMotor(MotorConfig.IntakeMotor2);
   }
+
+  public IntakeSubsystem(TimeOfFlightSensor timeOfFlightSensor) {
+    isConeMode = true;
+    this.timeOfFlightSensor = timeOfFlightSensor;
+
+    motor = MotorController.constructMotor(MotorConfig.IntakeMotor1);
+    motor2 = MotorController.constructMotor(MotorConfig.IntakeMotor2);
+  }
```

Comment:
This should call the no-argument version of the constructor, only initializing the TOF after